autoload/lsp/omni.vim

```diff
@@ -37,11 +37,10 @@ let s:completion_status_failed = 'failed'
 let s:completion_status_pending = 'pending'
 
 let s:is_user_data_support = has('patch-8.0.1493')
-let s:user_data_key = 'vim-lsp/textEdit'
-let s:user_data_additional_edits_key = 'vim-lsp/additionalTextEdits'
 let s:user_data_insert_start_key = 'vim-lsp/insertStart'
-let s:user_data_insert_format_key = 'vim-lsp/insertFormat'
 let s:user_data_filtertext_key = 'vim-lsp/filterText'
+let s:user_data_server_name_key = 'vim-lsp/serverName'
+let s:user_data_completion_item_key = 'vim-lsp/completionItem'
 
```

Comment:
textEdit/additionalTextEdits/insertFormat are used by CompleteDone.
New implementation provides `completionItem` instead.

Reply:
On purpose, I didn't change the `omnifunc` behavior in this PR.
But I think you are advice is right. those keys can removed.